services/api-server/tests/mocks/create_study_job.json

```diff
@@ -14,9 +14,9 @@
    "data": {
     "task_id": "POST%20%2Fv0%2Fprojects%3Ffrom_study%3Dfa07a3ec-f190-11ee-8b7b-0242ac140026%26hidden%3Dtrue.a02d7664-93cb-4a6f-917c-ac5ff06438fe",
     "task_name": "POST /v0/projects?from_study=fa07a3ec-f190-11ee-8b7b-0242ac140026&hidden=true",
-    "status_href": "/v0/tasks/POST%2520%252Fv0%252Fprojects%253Ffrom_study%253Dfa07a3ec-f190-11ee-8b7b-0242ac140026%2526hidden%253Dtrue.a02d7664-93cb-4a6f-917c-ac5ff06438fe",
-    "result_href": "/v0/tasks/POST%2520%252Fv0%252Fprojects%253Ffrom_study%253Dfa07a3ec-f190-11ee-8b7b-0242ac140026%2526hidden%253Dtrue.a02d7664-93cb-4a6f-917c-ac5ff06438fe/result",
-    "abort_href": "/v0/tasks/POST%2520%252Fv0%252Fprojects%253Ffrom_study%253Dfa07a3ec-f190-11ee-8b7b-0242ac140026%2526hidden%253Dtrue.a02d7664-93cb-4a6f-917c-ac5ff06438fe"
+    "status_href": "http://webserver:8080/v0/tasks/POST%2520%252Fv0%252Fprojects%253Ffrom_study%253Dfa07a3ec-f190-11ee-8b7b-0242ac140026%2526hidden%253Dtrue.a02d7664-93cb-4a6f-917c-ac5ff06438fe",
```

Comment:
Q: why do I see a DNS and not a IP here?

Reply:
This is because the way these unit tests are setup. the webserver is automatically mocked using here. But this is only done if I use `http://webserver:8000`